addons/web/static/tests/search/custom_filter_item_tests.js

```diff
@@ -441,6 +441,37 @@ QUnit.module("Search", (hooks) => {
         }
     );
 
+    QUnit.test("custom filter date with equal operator", async function (assert) {
+        assert.expect(2);
+
+        const originalZoneName = luxon.Settings.defaultZoneName;
+        luxon.Settings.defaultZoneName = new luxon.FixedOffsetZone.instance(-240);
+        registerCleanup(() => {
+            luxon.Settings.defaultZoneName = originalZoneName;
+        });
+
+        patchDate(2017, 1, 22, 12, 30, 0);
+
+        const controlPanel = await makeWithSearch({
+            serverData,
+            resModel: "foo",
+            Component: ControlPanel,
+            searchViewId: false,
+            searchMenuTypes: ["filter"],
+        });
+
+        await toggleFilterMenu(controlPanel);
+        await toggleAddCustomFilter(controlPanel);
+
+        await editConditionField(controlPanel, 0, "date_field");
+        await editConditionOperator(controlPanel, 0, "=");
+        await editConditionValue(controlPanel, 0, "01/01/2017");
+        await applyFilter(controlPanel);
+
+        assert.deepEqual(getFacetTexts(controlPanel), ['A date is equal to "01/01/2017"']);
```

Comment:
@Jinjiu96 you should replace controlPanel by target in all of these
IIRC only the below line (getDomain) should still use controlPanel

Reply:
Thank you very much! I will change that! :pray: